zookeeper-server/zookeeper-server-3.5.6/src/main/java/com/yahoo/vespa/zookeeper/LoggingWatcher.java

```diff
@@ -0,0 +1,19 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.zookeeper;
+
+import org.apache.zookeeper.WatchedEvent;
+import org.apache.zookeeper.Watcher;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+class LoggingWatcher implements Watcher {
```

Comment:
Consider inlining this as `(event) -> log.log(Level.INFO, event.toString())` so we don't need to have this class in every versioned module.

Reply:
Yes, good point, fixed.